Error while checking build/O2/fullCI for 0a0b803976e23144ec2881b65c12492c43b3f0c2 at 2023-03-15 06:20:
```
## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
/sw/SOURCES/O2/10951-slc8_x86-64/0//Framework/Core/include/Framework/DeviceStateEnums.h:1:1: error: missing or malformed copyright notice
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10951/0a0b803976e23144ec2881b65c12492c43b3f0c2/build_O2_fullCI/pretty.html).

--- a/Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h
+++ b/Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h
@@ -130,7 +130,7 @@ class MatchITSTPCQC
                                                               // with that label so far, and the flag to say if it is a physical primary or not
   o2::steer::MCKinematicsReader mcReader;                     // reader of MC information
 
-  // Pt
+  // Pt.
   TH1D* mPt = nullptr;
   TH1D* mPtTPC = nullptr;
   TEfficiency* mFractionITSTPCmatch = nullptr;

--- a/Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h
+++ b/Detectors/GlobalTracking/include/GlobalTracking/MatchITSTPCQC.h
@@ -16,6 +16,7 @@
 #ifndef ALICEO2_GLOBTRACKING_MATCHTPCITS_QC_
 #define ALICEO2_GLOBTRACKING_MATCHTPCITS_QC_
 
+#include <TH1D.h>
 #include <TH1F.h>
 #include <TH2F.h>
 #include <TEfficiency.h>

